sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/CountMinSketchAgg.scala

```diff
@@ -37,15 +37,22 @@ import org.apache.spark.util.sketch.CountMinSketch
  * @param confidenceExpression confidence, must be positive and less than 1.0
  * @param seedExpression random seed
  */
+// scalastyle:off nonascii
 @ExpressionDescription(
   usage = """
     _FUNC_(col, eps, confidence, seed) - Returns a count-min sketch of a column with the given esp,
       confidence and seed. The result is an array of bytes, which can be deserialized to a
       `CountMinSketch` before usage. Count-min sketch is a probabilistic data structure used for
       cardinality estimation using sub-linear space.
   """,
+  examples = """
+    Examples:
+      > SELECT _FUNC_(col, 0.1d, 0.9d, 0) FROM VALUES (1), (2), (1) AS tab(col);
+       ]�Z0jl����M��_
+  """,
```

Comment:
Is there a better way how to format this example?

Reply:
I think you can pick a number that produces smaller binary with less accuracy and higher error rate and use `hex` to show e.g.)

```
SELECT hex(count_min_sketch(col, 0.5d, 0.5d, 1)) FROM VALUES (1), (2), (1) AS tab(col)
0000000100000000000000030000000100000004000000005D8D6AB90000000000000000000000000000000200000000000000010000000000000000
```

Reply:
cc @viirya FYI

Reply:
+1